.github/actions/action.yml

```diff
@@ -0,0 +1,101 @@
+name: Setup Elixir Project
```

Comment:
I think `elixir-setup.yml` would be a more suitable name

Reply:
I can't, this file needs to be named "action.yml" specifically due to a convention established by GitHub Actions.